packages/map-toolkit/src/cursor-latlng/index.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { useOn } from '@accelint/bus/react';
+import { coordinateSystems, createCoordinate } from '@accelint/geo';
+import { useState } from 'react';
+import { MapEvents } from '../deckgl/base-map/events';
+import type {
+  MapHoverEvent,
+  MapViewStateEvent,
+} from '../deckgl/base-map/types';
+
+export type FormatTypes = 'dd' | 'ddm' | 'dms' | 'mgrs' | 'utm';
```

Comment:
up for debate: this should be in the types file I think, and could be updated to be an `as const` object for harder core type safety.